### DISCUSSION TITLE
Requested a review from doesn't show `a team`

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#121209] [Mobile] [answered]
GitHub Mobile app beta for Android 

https://github.com/actions/starter-workflows/pull/2107

App
![Screenshot_20240426-070551.png](https://github.com/community/community/assets/2119212/8835fe55-1281-48a1-8d43-7df6972548c0)

Browser
![Screenshot_20240426-070745.png](https://github.com/community/community/assets/2119212/5b4fade1-5eae-4ed6-9c22-171f5ed8fb98)

**Comment by HenSquared** [comment, marked as answer]
Thanks @jsoref . I've created an Issue in our Android repository to track this issue.
